samza-core/src/main/java/org/apache/samza/clustermanager/JobCoordinatorLaunchUtil.java

```diff
@@ -19,27 +19,36 @@
 package org.apache.samza.clustermanager;
 
 import java.util.List;
+import java.util.Map;
+import com.google.common.annotations.VisibleForTesting;
 import org.apache.samza.SamzaException;
 import org.apache.samza.application.SamzaApplication;
 import org.apache.samza.application.descriptors.ApplicationDescriptor;
 import org.apache.samza.application.descriptors.ApplicationDescriptorImpl;
 import org.apache.samza.application.descriptors.ApplicationDescriptorUtil;
 import org.apache.samza.config.Config;
 import org.apache.samza.config.JobConfig;
+import org.apache.samza.config.JobCoordinatorConfig;
 import org.apache.samza.config.MapConfig;
+import org.apache.samza.config.MetricsConfig;
+import org.apache.samza.coordinator.StaticResourceJobCoordinator;
 import org.apache.samza.coordinator.metadatastore.CoordinatorStreamStore;
 import org.apache.samza.execution.RemoteJobPlanner;
 import org.apache.samza.metadatastore.MetadataStore;
 import org.apache.samza.metrics.MetricsRegistryMap;
+import org.apache.samza.metrics.MetricsReporter;
 import org.apache.samza.util.CoordinatorStreamUtil;
 import org.apache.samza.util.DiagnosticsUtil;
+import org.apache.samza.util.MetricsReporterLoader;
 
 
 /**
  * Util class to launch and run {@link ClusterBasedJobCoordinator}.
  * This util is being used by both high/low and beam API Samza jobs.
  */
 public class JobCoordinatorLaunchUtil {
+  private static final String JOB_COORDINATOR_CONTAINER_NAME = "JobCoordinator";
```

Comment:
When starting job coordinator jvm process, we pass `-Dsamza.container.name=samza-job-coordinator` jvm parameter explicitly in `run-jc.sh` script. 

Looks like here is the inconsistency part. Can you confirm if we should using the same container name?

Reply:
Looks like the logging function requires this variable `samza.container.name`: https://github.com/apache/samza/blob/master/docs/learn/documentation/versioned/jobs/logging.md
 
Can you confirm if we still need the variable?